recipes/libcurl/all/conanfile.py

```diff
@@ -288,26 +299,16 @@ def _configure_autotools_vars(self):
         return autotools_vars
 
     def _configure_autotools(self):
-        if not self._autotools:
-            use_win_bash = self._is_mingw and not tools.cross_building(self.settings)
-            self._autotools = AutoToolsBuildEnvironment(self, win_bash=use_win_bash)
+        autotools_vars = self._configure_autotools_vars()
+        if self._autotools:
+            return self._autotools, autotools_vars
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
 
-            if self.settings.os != "Windows":
-                self._autotools.fpic = self.options.fPIC
-
-            autotools_vars = self._configure_autotools_vars()
-
-            # tweaks for mingw
-            if self._is_mingw:
-                # patch autotools files
-                self._patch_mingw_files()
-
-                self._autotools.defines.append('_AMD64_')
-
-            configure_args = self._get_configure_command_args()
-            self._autotools.configure(vars=autotools_vars, args=configure_args)
-
-        return self._autotools, self._configure_autotools_vars()
+        configure_args = self._get_configure_command_args()
+        if self._is_mingw:
+            self._autotools.defines.append('_AMD64_')
```

Comment:
maybe it needs a check for **x86_64** arch?

Reply:
ping @madebr

Reply:
Indeed, in the sources it is also only added when w64 is detected.